### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by Baayeh**
## Controllers, Actions, Routes and Views

> In this PR, I completed the following tasks:

- [x] For each URL from the wireframes, I created a `controller`, an `action` in the correct `controller`,  `route` and a `view` file.
- [x] View files included only placeholders written in plain HTML.
- [x] Made sure that I added code to handle only the 4 required URLs from the wireframes.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  shallow do
+    resources :users, only: [:index, :show] do
+      resources :posts, only: [:index]
```
Comment (mikethreels) [inline config/routes.rb]: - [x] in this case, for the posts only the index endpoint is available please make sure the show endpoint is also available.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Steps, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (Baayeh) [reply config/routes.rb]: Thanks for the observation. It might have escaped me.

**Review by mikethreels — APPROVED** [review]
Hi Steps, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**